### ISSUE TITLE
crash in +[NSData(ImageContentType) sd_imageFormatForImageData:] 

### ISSUE THREAD

**Issue by happyxb** [SDWebImage/SDWebImage#2804]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 12.4
 SDWebImage Version      | 4.3.3
 Integration Method      | cocoapods
 Xcode Version           |  Xcode 10
 Repro rate              | one device muti times
 Device                 | iPhone 6 Plus
 

网络APN |   | 是否越狱 | 否
-- | -- | -- | --
可用内存大小 | 19.32 MB ( 1.99% ) | 可用存储空间 | 107.34 GB ( 90.05% )
Bugly SDK版本号 | 2.5.0

### Issue Description and Steps

```
#220737 SIGBUS
BUS_ADRALN
+[NSData(ImageContentType) sd_imageFormatForImageData:] (NSData+ImageContentType.m:32)
-[SDWebImageWebPCoder canDecodeFromData:] (SDWebImageWebPCoder.m:50)
-[SDWebImageCodersManager decodedImageWithData:] (SDWebImageCodersManager.m:107)
-[SDImageCache diskImageForKey:data:options:] (SDImageCache.m:0)
-[SDImageCache diskImageForKey:data:] (SDImageCache.m:0)
-[SDImageCache diskImageForKey:] (SDImageCache.m:475)
-[SDImageCache imageFromDiskCacheForKey:] (SDImageCache.m:0)
-[SDImageCache imageFromCacheForKey:] (SDImageCache.m:0)
```

**Comment by kinarobin** [comment]
@happyxb Could you give more detail crash info for this issue?

**Comment by dreampiggy** [comment]
**one device muti times**

Can you get that device ? Then you can using debugger and set a exception breakpoint and run.

I think it's something issue about the image data or memory issue.

**Comment by happyxb** [comment]
the device available memory only 19.32MB,maybe this is the reason.
I don't have any more info about the device or the image data.

**Comment by dreampiggy** [comment]
@happyxb  Any reply for this issue, a full stacktrace can be used to investigate this issue ?
